### DISCUSSION TITLE
关于训练，怎么模型并行？

### DISCUSSION THREAD

**Discussion by CNUIGB** [zai-org/ChatGLM3#613] [General]
在训练过程中，官方给出的ptuningv2训练脚本应该是数据并行脚本，即将模型的参数复制到num_gpus张GPU上，然后在将example分配给每个GPU并行训练。
但是想问一下怎么实现模型并行+数据并行，即将模型拆分加载到多张GPU上，然后再数据并行？

**Comment by zRzRzRzRzRzRzR** [comment]
#253
